watertap/property_models/cryst_prop_pack.py

```diff
@@ -1116,7 +1116,7 @@ def _make_state_vars(self):
         self.temperature = Var(
             domain=NonNegativeReals,
             initialize=298.15,
-            bounds=(273.15, 373.15),
+            bounds=(273.15, 500),  # 373.15),
```

Comment:
```suggestion
            bounds=(273.15, 500),
```
Why did you increase this upper bound? We should make sure the properties are valid in this upper range before changing.